src/Microsoft.AspNetCore.Mvc.Core/Internal/DefaultModelBindingContext.cs

```diff
@@ -130,32 +132,16 @@ public DefaultModelBindingContext()
         }
 
         /// <inheritdoc />
-        public override NestedScope EnterNestedScope()
```

Comment:
This wasn't an important part of this PR but you haven't actually fixed anything yet. Either revert this part of the change or move `CreateBindingContext()` and the other `EnterNestedScope()` after the properties as well.